tests/em/tdem/test_TDEM_MultiSrc_DerivAdjoint.py

```diff
@@ -3,12 +3,7 @@
 import numpy as np
 from SimPEG import Mesh, Maps, SolverLU, Tests
 from SimPEG import EM
-
-try:
-    from pymatsolver import PardisoSolver
-    Solver = PardisoSolver
-except ImportError:
-    Solver = SolverLU
+from pymatsolver import Pardiso as Solver
```

Comment:
TDEM tests need to run on Pardiso, otherwise it takes way too long. This is also a good check that Pardiso is being properly installed on travis